.env.sample

```diff
@@ -7,7 +7,12 @@ SERVICE_ADMIN_PK=
 
 ETH_RPC_URL=
 POLYGON_RPC_URL=
+ARBITRUM_ALCHEMY_RPC_URL=
+ARBITRUM_ANKR_RPC_URL=
```

Comment:
do we need multiple URLs for this?

can we just do `ARBITRUM_RPC_URL`?

either that, or we should do:
- ARBITRUM_RPC_URL_ALCHEMY
- ARBITRUM_RPC_URL_ANKR
- ARBITRUM_RPC_PICKER = ALCHEMY || ANKR